src/ws/service/friendships_service.rs

```diff
@@ -95,11 +96,12 @@ impl FriendshipsServiceServer<SocialContext> for MyFriendshipsService {
                     }
                 });
 
+                log::info!("Returning generator for all friends");
```

Comment:
I think it'd be useful to add the user id here, it'll help us to track the flow of a given user much more easily. wdyt?
```suggestion
                log::info!("Returning generator for all friends for user {}", user_id.social_id);
```